### PR TITLE
Feature/nextflow upgrade 23 10 0

### PR DESCRIPTION
**Pull request by BasMonkey**
• Added allow overwrites to the config so that a workflow can be restarted.
• Refactor of the in- and output variables so that the workflows can be run with Nextflow 23.10.0 (mainly in CustomModules, [See the CustomModules repo](https://github.com/UMCUGenetics/CustomModules/tree/feature/nextflow_upgrade_23_10_0))

### CODE REVIEW

File: run_nextflow_wes.sh [hunk 1]
```diff
@@ -15,6 +15,27 @@ mkdir -p log
 if ! { [ -f 'workflow.running' ] || [ -f 'workflow.done' ] || [ -f 'workflow.failed' ]; }; then
 touch workflow.running
 
+file="log/nextflow_trace.txt"
```
Comment (ellendejong) [inline run_nextflow_wes.sh:18]: You should use full path instead.

File: run_nextflow_wes.sh [hunk 2]
```diff
@@ -15,6 +15,27 @@ mkdir -p log
...
+
+if [ -e "$file" ]; then
+    # Extract the current suffix from the file name
+    current_suffix=$(echo "$file" | grep -oE '[0-9]+$')
```
Comment (ellendejong) [inline run_nextflow_wes.sh:22]: My personal preference: using curly brackets if variables.

File: run_nextflow_wes.sh [hunk 3]
```diff
@@ -15,6 +15,27 @@ mkdir -p log
...
+    if [ -z "$current_suffix" ]; then
+        # If no suffix found, set it to 0
+        current_suffix=0
+    fi
+
+    # Increment the suffix
+    new_suffix=$((current_suffix + 1))
+
+    # Create the new file name with the incremented suffix
+    new_file="${file%.*}_$new_suffix.${file##*.}"
```
Comment (ellendejong) [inline run_nextflow_wes.sh:33]: Nice, I will remember this notation. Norally I use `basename` instead, but think I would prefer this alternative :)

**Review by ellendejong — CHANGES_REQUESTED** [review]

**Review by ellendejong — DISMISSED** [review]

**Review by melferink — APPROVED** [review]
Ok!

**Pull request merged**